zawata/S19-Team2
# Flask server

Contains the server portion.

To test:

1. `parcel build web/index.html` from root
2. `cmake --build build --target all` from root (may have to create build folder)
3. use python to run FlaskServer.py
4. In browser, open http://loaclhost:5000/web/index.html to view simulation
5. Download one of the test bsp files from google drive, and then visit http://loaclhost:5000/data.trajectory to upload it to the server.

You should get some debug messages, but no complaints (other than maybe a missing icon file)
